data/strings/strings.txt

```diff
@@ -9847,12 +9847,82 @@
     comment = Tab title in the time picker
     tags = android
     en = From
+    ar = من الساعة
+    be = З
+    bg = От
+    ca = De
+    cs = Od
+    da = Fra
+    de = Von
+    el = Από τις
+    es = De
+    et = Kell
+    eu = etatik
+    fa = از ساعت
+    fi = alkaen
+    fr = De
+    he = משעה 6:00 עד 11:00
+    hu = tól
+    id = Dari pukul
+    it = Dalle
+    ja = から
+    ko = 부터
+    mr = ते
+    nb = Fra
+    nl = Van
     pl = Od
+    pt = Das
+    pt-BR = Das
```

Comment:
```suggestion
```

Reply:
Duplicate sub-regional languages are not needed if they are the same with base ones (pt).

Reply:
Fixed.